tools/quantp/quantp.xml

```diff
@@ -0,0 +1,131 @@
+                    <option value="none">None</option>
+                </param>
+                <param name="volcano_with" type="select" display="radio" label="Volcano plot with p-value or adjusted p-value">
+                    <option value="pval" selected="True">P-value (Default)</option>
+                    <option value="adj_pval">Adjusted P-value</option>
+                </param>
+            </when>
+        </conditional>
+        <param name="cooksd_upper" type="integer" value="4" optional="false" >
+            <label>Influential Observation cutoff: Observations > value * mean of Cook's distances (Default: "4" * mean(Cook's Distance))</label>
+        </param>
+        <param name="nclust" type="select" label="K-mean clustering: Number of clusters">
+            <option value="1">1</option>
+            <option value="2">2</option>
+            <option value="3">3</option>
+            <option value="4" selected="True">4</option>
+            <option value="5">5</option>
+            <option value="6">6</option>
+            <option value="7">7</option>
+            <option value="8">8</option>
+            <option value="9">9</option>
+            <option value="10">10</option>
+        </param>
+      
+        <param name="hm_nclust" type="select" label="Hierarchical clustering: Number of clusters (from Heatmap)">
+            <option value="1">1</option>
+            <option value="2">2</option>
+            <option value="3">3</option>
+            <option value="4">4</option>
+            <option value="5" selected="True">5</option>
+            <option value="6">6</option>
+            <option value="7">7</option>
+            <option value="8">8</option>
+            <option value="9">9</option>
+            <option value="10">10</option>
+        </param>
+    </inputs>
+    <outputs>
+        <data format="html" name="html_file" label="protein_transcript_correlation on ${pe_exp.name} and ${te_exp.name}.html"/>
+    </outputs>
+    <tests>
+        <test>
+            <conditional name="experiment_design_option">
+                <param name="sample_type" value="multiple"/>
+                <param name="method_type" value="mean"/>
+                <param name="exp_design" value="exp_design_file.tabular"/>
+                <param name="correction_method" value="bonferroni"/>
+                <param name="volcano_with" value="pval"/>
+            </conditional>
+            <param name="pe_exp" value="protein_data.tabular"/>
+            <param name="te_exp" value="transcript_Data.tabular"/>
+            <param name="cooksd_upper" value="4"/>
+            <param name="nclust" value="5"/>
+            <param name="hm_nclust" value="5"/>
+            <output name="html_file" file="protein_transcript_correlation.html"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+**QuanTP: Correlation between protein and transcript abundances**
+
+Developer: Praveen Kumar and Priyabrata Panigrahi
+
+
+OUTPUT:
```

Comment:
This can also be removed if you do not explain what a user can see. Many users might not even know what HTML is ...